extensions/packageManager/readme.md

```diff
@@ -1,144 +1,155 @@
+The output of the CLI tool will include a list of the files that were added, deleted or updated. Note that **changes to existing files will be overwritten if newer versions are found in a package.**
 
-# How to build your own packages
+## Build your own packages
 
-The packages used by Composer are native packages - the process of building one is the same as it would be for any software component published to NPM or Nuget.
+The packages used by Composer are native packages, and the process of building one is the same as it would be for any software component published to NPM or NuGet.
 
-However, the packages that we care about will contain one of more of the following elements:
+However, the packages that we care about should contain one or more of the following elements:
 
-* a .schema file describing an Adaptive Dialog $kind. <a href="https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-dialogs-declarative?view=azure-bot-service-4.0">Learn more</a>
+* a `.schema` file describing an Adaptive Dialog `$kind`. <a href="https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-dialogs-declarative?view=azure-bot-service-4.0">Learn more about schema file and how to create one.</a>
 * an `exported` folder containing one or more sets of dialog files
 * an `exported` folder containing one or more sets of LG files
 
-## How to package a custom action
-
-1. <a href="https://docs.microsoft.com/en-us/composer/how-to-add-custom-action">Follow these steps</a> to create the code and schema for your custom action. Both the code and schema files are required to create the package.
-2. Optionally add additional dialogs to the bundle using the instructions below.
-3. Follow the "For Nuget" instructions below.
+### Package dialogs
 
-## How to package dialogs for re-use
+1. Open your bot project and find the `dialogs` folder. The files for each dialog are contained in a named sub-folder here. Identify the dialogs you want to share. Please note that you should not package the main dialog.
+2. Create a new folder outside the bot project folder with the name of your new package. For example, `cool-dialogs`.
+3. Inside this new folder, create a subfolder called `exported`.
+4. Copy the dialogs you want to share from the original `dialogs` folder into the new `exported` folder.
+5. Follow the instructions ([NuGet](#for-nuget) |[NPM](#for-npm)) below to publish the package to NuGet and/or NPM.
 
-To share dialogs you've created in Composer:
+### Package LG files
 
-1. Open your bot project and find the `dialogs` folder. The files for each dialog are contained in a named sub-folder here.  Identify the dialogs you want to share.
-2. Create a new folder with the name of your new package. For example, `cool-dialogs`
+1. Open your bot project and find the `language-generation` folder that contains the LG files you want to share.
```

Comment:
There is a little bit more complexity here.

Inside language-generation, there will be locale folders like `en-us` -- and then inside these, there will be the actual LG files.

To support multiple languages, the structure should retain the locale subfolders -

so you might end up where you have 2 LG files in 2 languages, for example:

exported/en-us/fancy.en-us.lg
exported/en-us/cool.en-us.lg
exported/de-de/fancy.de-de.lg
exported/de-de/cool.de-de.lg

Reply:
I updated the wording to clarify this.